migrations/probo-asset-fix-and-upload

```diff
@@ -0,0 +1,104 @@
+#! /usr/bin/env node
```

Comment:
Can we rename the migrations to include a number indicating the order they need to be run? e.g. `0001-probo-asset-fix-and-upload`